package.json

```diff
@@ -7,6 +7,13 @@
   "main": "dist/lib/auth0-vue.cjs.js",
   "types": "dist/typings/index.d.ts",
   "module": "dist/auth0-vue.production.esm.js",
+  "exports": {
+    ".": {
```

Comment:
Is `.` necessary if there are no other subpaths?

Reply:
I think leaving it open to extend in future is best as I guess any changes to this would be classed as a breaking change.

I wonder if there's some value in also includes a `node` stanza with `production` and `development` variants like [vue-router does](https://github.com/vuejs/router/blob/main/packages/router/package.json#L10-L25)? Not sure if that would improve the development experience in regards to errors?

Reply:
That would involve changing the bundles we generate, as we don't have both dev and prod bundles for CJS and ESM. Would it make sense to do this in a follow up PR?

Reply:
Yeah that's fair to keep this PR focused

Reply:
As mentioned on slack, going to hold of on these changes because of the fact that I am currently a bit occupied. Going to cut a release that includes the changes from this PR, as we can add the additional changes in a non breaking way when time allows.